tensorflow/compiler/xrt/kernels/xrt_execute_op.cc

```diff
@@ -388,24 +391,24 @@ xla::StatusOr<RefPtr<XRTTupleAllocation>> ExecuteComputation(
   // memory, until either the runfn can run, or we run out of freeable memory.
   return memory_manager->Run<RefPtr<XRTTupleAllocation>>(
       runfn, device_ref->backend(), device_ref->device_ordinal(),
-      /*requested_free_size=*/0);
+      /*requested_free_size=*/0, allocator);
```

Comment:
Instead of passing `allocator` all the way down here, couldn't we call `device_ref->GetMemoryAllocator()` directly?

Reply:
Done.